software/database/src/main/java/org/apache/brooklyn/entity/database/postgresql/PostgreSqlNode.java

```diff
@@ -52,13 +52,17 @@
 public interface PostgreSqlNode extends SoftwareProcess, HasShortName, DatastoreCommon, DatabaseNode {
     
     @SetFromFlag("version")
-    ConfigKey<String> SUGGESTED_VERSION = ConfigKeys.newConfigKeyWithDefault(SoftwareProcess.SUGGESTED_VERSION, "9.3-1");//"9.1-4");
+    ConfigKey<String> SUGGESTED_VERSION = ConfigKeys.newConfigKeyWithDefault(SoftwareProcess.SUGGESTED_VERSION, "9.3-3");//"9.1-4");
 
     @SetFromFlag("configFileUrl")
     ConfigKey<String> CONFIGURATION_FILE_URL = ConfigKeys.newStringConfigKey(
             "postgresql.config.file.url", "URL where PostgreSQL configuration file can be found; "
                 + "if not supplied the blueprint uses the default and customises it");
 
+    @SetFromFlag("curlCommand")
+    ConfigKey<String> CURL_COMMAND = ConfigKeys.newStringConfigKey(
+            "curl.command", "cURL comamnd to be used. It follows location redirects by default (added -L parameter)", "curl -L");
```

Comment:
+1 could even do away with the config altogether - that's not something I've ever seen needing configuration. No strong opinion though.